Neos.Neos/webpack.styles.config.js

```diff
@@ -0,0 +1,98 @@
+const debug = process.env.NODE_ENV !== "production";
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const ImageMinimizerPlugin = require("image-minimizer-webpack-plugin");
+const TerserPlugin = require("terser-webpack-plugin");
+const IgnoreEmitPlugin = require("ignore-emit-webpack-plugin");
+
+const stylesConfig = {
+	context: __dirname,
+	devtool: !debug ? "source-map" : false,
+	entry: {
+		Main: ["./Resources/Private/Styles/Neos.scss"],
+		Lite: ["./Resources/Private/Styles/Lite.scss"],
+		Minimal: ["./Resources/Private/Styles/Minimal.scss"],
+		Login: ["./Resources/Private/Styles/Login.scss"],
+		Error: ["./Resources/Private/Styles/Error.scss"],
+		RawContentMode: ["./Resources/Private/Styles/RawContentMode.scss"],
+		Welcome: ["./Resources/Private/Styles/Welcome.scss"],
+	},
+	output: {
+		path: __dirname + "/Resources/Public/Styles",
+		filename: "[name].js",
+	},
+	resolve: {
+		extensions: [".css", ".scss", ".sass"],
+	},
+	module: {
+		rules: [
+			{
+				test: /\.(png|gif|jpe?g|svg)$/i,
+				include: __dirname + "/Resources/Public/Images/",
+				use: [
+					{
+						loader: "url-loader",
+						options: {
+							name: "images/[name].[ext]",
+							publicPath: "../",
+							limit: 30000,
+						},
+					},
+				],
+			},
+			{
+				test: /\.scss$/,
+				use: [MiniCssExtractPlugin.loader, "css-loader", "sass-loader"],
+			},
+			{
+				test: /\.(eot|ttf|woff|woff2)$/,
+				use: [
+					{
+						loader: "url-loader",
+						options: {
+							name: "[name].[ext]",
+							publicPath: "../Fonts/",
+						},
+					},
+				],
+			},
+		],
+	},
+	plugins: [
+		new MiniCssExtractPlugin({
+			filename: "[name].css",
+		}),
+		new IgnoreEmitPlugin(/([a-zA-Z0-9\s_\\.\-\(\):])+(.js|.map.js)$/),
+		new ImageMinimizerPlugin({
```

Comment:
Is it worth to have a minimizer step for those few images we have there?
We could optimise them once and just keep it that way.

Reply:
But with that step we can never forget it when we add new assets in future. And maybe a contributor don't know that we minify assets and so we can forget about this.

Reply:
But the libraries you have there don't really minify.

Reply:
You mean the library folder in Javascript? Because this will be gone without the emberjs stuff I guess.

Reply:
No jpegtran and the others don't really minify that much. They remove metadata.